src/portals/blockchain-frontend/apps/blockchain-frontend/components/auth/Auth.tsx

```diff
@@ -0,0 +1,64 @@
+import styles from './Auth.module.css';
+import React, { useState } from 'react';
+import Login from '../login/Login';
+import SignUp from '../signup/Signup';
+//import { useComponentState } from './state';
+
+export default function Auth() {
+  const [isRegister, setIsRegister] = useState(false);
+
+  const toggleRegister = () => {
+    setIsRegister(!isRegister);
+  };
+  return (
+    <div className={styles['container']}>
+      <div className={styles['main-login']}>
+        <div className={styles['login-contain']}>
+          <div className={styles['left-side']}>
+            {isRegister ? <SignUp /> : <Login />}
+            <div className={styles['toggle-text']}>
+              {isRegister ? (
```

Comment:
don't need to render a whole block here. The only difference of two blocks are the labels. Just render labels conditionally.
{isRegister ? Login here : Sign up now}